playbooks/roles/deploy-osh/tasks/main.yml

```diff
@@ -417,3 +417,20 @@
   tags:
     - heat
     - run
+
+- name: Deploy Tempest
+  import_tasks: component-install.yml
+  vars:
+    _component_runcommand: "./tools/deployment/multinode/900-tempest.sh"
+    _component_run_env:
+      OSH_EXTRA_HELM_ARGS_HEAT: >
+        --values=/tmp/socok8s-susedefaults-tempest.yaml
+        --values=/tmp/socok8s-useroverrides-tempest.yaml
+        {{ lookup('env', 'OSH_EXTRA_HELM_ARGS_TEMPEST') | default('', True) }}
+    _component_overrides:
+      - suse_file: tempest.yaml
+        user_overrides: "{{ suse_osh_deploy_tempest_yaml_overrides }}"
+  tags:
+    - tempest
+    - run
+  when: deploy_tempest
```

Comment:
I would do ``(deploy_tempest | default(False)) | bool``

or add `deploy_tempest` in `defaults` to `False`.

The user can then change the extravars to alter deployment.

Reply:
sounds good to me, will rework it for using extravars instead

Reply:
Please note that if we add other defaults like deploy_cinder deploy_glance deploy_nova, we could use those variables to:
1) Decide to deploy those components
2) Decide to use those components in tempest based on their value.

Reply:
surely we're giving the default in the defaults/main.yml which is much more discoverable?